libs/pages/elewa/home/src/lib/components/home-job-section/home-job-section.component.scss

```diff
@@ -0,0 +1,24 @@
+.jobs-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    min-width: 100vw;
+    min-height: 100vh;
+    font-size: xx-large;
+}
+
+p {
+    color: var(--primary-color);
+    margin-bottom: 1rem;
+}
+a {
+    font-size: var(--font-size-lg);
+    font-weight: bold;
+    color: var(--secondary-color);
+    text-decoration: none;
+    
+}
+
+
```

Comment:
The a tag link should have an underline on hover (basic hyperlink features).
remove empty lines on this file.